configure.py

```diff
@@ -550,12 +546,6 @@
     help='do not use V8 includes from the bundled deps folder. ' +
          '(This mode is not officially supported for regular applications)')
 
-parser.add_option('--build-v8-with-gn',
```

Comment:
Please instead of delete move to ~https://github.com/nodejs/node/blob/5cb8b2547f025f12a89d34523a6191c2a8a080ed/configure.py#L470-L491
with `help=optparse.SUPPRESS_HELP`

Reply:
^^^
To avoid making this semver-major.